pytwis/pytwis_clt.py

```diff
@@ -221,96 +222,100 @@ def pytwis_command_parser(raw_command):
         
     Returns
     -------
-    command_with_args: list(str, dict(str, str or int))
-        The parsed command output. The first element of the list is the command type, e.g., 'register', 
-        and the second element is the command arguments, e.g., {'username': <username>, 'password': <password>}
-        for `register`.
-    
+    arg_dict: dict(str, str or int)
+        The parsed command output.
+        {'command':'register', 'username': <username>, 'password': <password>} for `register`.
     Raises
     ------
     ValueError
         If the raw command can't be parsed correctly, e.g., it has an incorrect number of arguments or 
         incorrect arguments.
     """
+    validate_command(raw_command)
+
+    # Some command (e.g., logout) may not have arguments.
     # Separate the command from its arguments.
     splited_raw_command = raw_command.split(' ', 1)
-    command_with_args = [splited_raw_command[0]]
 
-    # Some command (e.g., logout) may not have arguments.
     arg_dict = {}
 
-    validate_command(raw_command)
-
-    if command_with_args[0] == CmdConstant.CMD_REGISTER:
+    if splited_raw_command[0] == CmdConstant.CMD_REGISTER:
         # register must have two arguments: username and password.
         args = splited_raw_command[1]
-        arg_dict = parse.parse('{{{arg1}}} {{{arg2}}}'.format(arg1=CmdConstant.ARG_USERNAME, 
+
+        arg_dict = parse.parse('{{{arg1}}} {{{arg2}}}'.format(arg1=CmdConstant.ARG_USERNAME,
                                                               arg2=CmdConstant.ARG_PASSWORD), args)
         if arg_dict is None:
             raise ValueError('{} has incorrect arguments'.format(CmdConstant.CMD_REGISTER))
         elif ' ' in arg_dict[CmdConstant.ARG_PASSWORD]:
             raise ValueError("password can't contain spaces")
 
-        print('{}: username = {}, password = {}'.format(CmdConstant.CMD_REGISTER, 
-                                                        arg_dict[CmdConstant.ARG_USERNAME], 
+        arg_dict = arg_dict.named
```

Comment:
May I ask what "named" is?

Reply:
parse doesn't return a dictionary. 
https://pypi.python.org/pypi/parse

Reply:
Oh, I didn't know that. Thank you for answering my question!